slang-checks/src/test/resources/com/sonarsource/slang/checks/StringLiteralDuplicated.threshold_4.slang

```diff
@@ -0,0 +1,31 @@
+x = "string literal1"                           // Compliant - only appears 3 times which is less than the custom threshold of 4
```

Comment:
I would prefer that this file does not mostly duplicate StringLiteralDuplicated.slang.
I think it should only be used to test the threshold.